src/utilities/exceptions.py

```diff
@@ -1,2 +1,35 @@
 class DynamoDBError(Exception):
     """Exception class which represents a response error of DynamoDB"""
+
+
+class ValidationError(Exception):
+    """Exception class which represents a different story validation errors"""
+
+    def __init__(self, message):
+        self.message = message
+        super().__init__(message)
+
+
+class RootDoesNotExistValidationError(ValidationError):
+    def __init__(self, message="The provided root node does not exist"):
+        super().__init__(message)
```

Comment:
Suppose we don't want to change the message in initializer, since we extend ValudationError class and change the message.
```suggestion
    MESSAGE = "The provided root node does not exist"
    def __init__(self):
        super().__init__(self.MESSAGE)
```